crates/symex_lib/src/any.rs

```diff
@@ -1,46 +1,56 @@
 use super::symbolic;
-use std::mem::MaybeUninit;
 
+/// Provides the [`any`] operation on the implementor
+/// ---
```

Comment:
Haven't really seen line separators used like this, seems kind of unidiomatic for docs

Reply:
Looking at `cargo doc --open`, it generates nice headers that are separated out, it works in the same way as the main header as far as I know. If you prefer the style with `#` as the main rubric I can refactor it to use that instead, I think it should produce the same output.